cost/flexera/cco/scheduled_report_markupsdowns/README.md

```diff
@@ -45,11 +38,20 @@ Leave the field blank to report on all top level Billing Centers.
 - *IOT markup or markdown percentage* - markup for the IOT category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markup).
```

Comment:
Same as IoT comment in .spellignore file - I think this should be IoT rather than IOT

Reply:
This appears to be based on a category that appears in our platform as IOT